mm2src/trezor/src/utxo/utxo_command.rs

```diff
@@ -0,0 +1,69 @@
+use crate::client::TrezorClient;
+use crate::constants::TrezorCoin;
+use crate::proto::messages_bitcoin as proto_bitcoin;
+use crate::response_channel::{response_loop, trezor_response_channel, TrezorResponseReceiver};
+use crate::{ecdsa_curve_to_string, serialize_derivation_path, TrezorResult};
+use common::executor::spawn;
+use futures::future::{select, FutureExt};
+use hw_common::primitives::{DerivationPath, EcdsaCurve};
+
+// Bitcoin(UTXO) operations.
+impl TrezorClient {
```

Comment:
These are only UTXO operations because they require to send Protobuf messages from `messages_bitcoin.proto`.
I think it's better to add `get_eth_public_key` and others in the future.

Reply:
It's ok for now, please check if it can be made generic on the next iteration.